pallets/automation-time/src/mock.rs

```diff
@@ -556,9 +639,10 @@ pub fn add_task_to_missed_queue(
 	task_id: TaskIdV2,
 	scheduled_times: Vec<u64>,
 	action: ActionOf<Test>,
+	cancel_errors: Vec<String>,
```

Comment:
Should this `cancel_errors` in this test case be updated?

Reply:
This does not need to be removed. Because we need to record the abort_errors of each task in order to know how to terminate the task.

Reply:
Yes, Fixed!